images/kube-api-rewriter/local/Dockerfile

```
@@ -9,10 +12,10 @@ RUN go mod download
 ADD . /app
 
 RUN GOOS=linux \
-    go build -o proxy ./cmd/kube-api-proxy
+    go build -o kube-api-rewriter ./cmd/kube-api-rewriter
 
 # Go builder.
-FROM golang:1.21-alpine3.19 AS builder-test-controller
+FROM golang:1.22-alpine3.19 AS builder-test-controller
```

We need to use the version of the image containing a specific version of the go language to avoid downloading another version
```suggestion
FROM golang:1.22.7-alpine3.19 AS builder-test-controller
```